File: app/src/main/java/com/yasu_k/saezuri/SetAdView.java
package com.yasu_k.saezuri;

import android.content.Context;
import com.google.android.gms.ads.AdRequest;
import com.google.android.gms.ads.AdView;
import com.google.android.gms.ads.MobileAds;

public class SetAdView {
    AdView mAdView;
    Context mContext;

    public SetAdView(AdView adView, Context context){
        this.mAdView = adView;
        this.mContext = context;
    }

    public void initAd(){
        MobileAds.initialize(mContext, initializationStatus -> {
        });

        if (mAdView == null) {
            throw new IllegalArgumentException(
                    "Can't initialize with null AdView.");
        }

        //mAdView = findViewById(R.id.adView);
        AdRequest adRequest = new AdRequest.Builder().build();
        mAdView.loadAd(adRequest);
    }
}

File: app/src/main/java/com/yasunari_k/saezuri/SetAdView.kt
package com.yasunari_k.saezuri

import android.content.Context
import com.google.android.gms.ads.AdRequest
import com.google.android.gms.ads.AdView
import com.google.android.gms.ads.MobileAds
import com.google.android.gms.ads.initialization.InitializationStatus

class SetAdView(var mAdView: AdView?, var mContext: Context) {
    fun initAd() {
        MobileAds.initialize(mContext) { initializationStatus: InitializationStatus? -> }
        requireNotNull(mAdView) { "Can't initialize with null AdView." }

        //mAdView = findViewById(R.id.adView);
        val adRequest = AdRequest.Builder().build()
        mAdView!!.loadAd(adRequest)
    }
}